app/main.py

```diff
@@ -1,8 +1,20 @@
 class Person:
-    # write your code here
-    pass
+    people = dict()
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    people_list = []
+    for person in people:
+        people_list.append(Person(person["name"], person["age"]))
+    for person in people:
+        if "wife" in person and person["wife"] is not None:
+            Person.people[person["name"]].wife = Person.people[person["wife"]]
+        elif "husband" in person and person["husband"] is not None:
+            Person.people[person["name"]].husband = \
+                Person.people[person["husband"]]
```

Comment:
Write inline or do as:
```suggestion
            husband = Person.people[person["husband"]]
            Person.people[person["name"]].husband = husband
```